Frontend/src/views/Account.vue

```diff
@@ -63,11 +64,24 @@ export default {
         message: "",
         user: {},
         loading: false,
+        logging: {
+          webpage: '',
+          webpageDurationStart: 0,
+        }
       }
   },
   created() {
+    // Browser logger, listener if used switches/closes tab
+    document.addEventListener('beforeunload', this.userBrowserTabSession)
+    this.logging.webpage = this.$options.name;
```

Comment:
$options is the export default I take it?

Reply:
yes